CHANGELOG.md

```diff
@@ -1,3 +1,169 @@
+Version 1.15.0
+==============
+**July 6, 2022**
+
+This is a major release with a large number of new features and bug
+fixes, as well as a updates to Python and Panel compatibility.
+
+Many thanks to the numerous users who filed bug reports, tested
+development versions, and contributed a number of new features and bug
+fixes, including special thanks to @ablythed @douglas-raillard-arm
+@FloLangenfeld @HoxBro @ianthomas23 @jenssss @pepijndevos @peterroelants
+@stas-sl @Yura52 for their contributions. In addition, thanks to the
+maintainers @jbednar, @maximlt, @jlstevens and @philippjfr for
+contributing to this release.
+
+Compatibility:
+
+- Python 2 support has finally been dropped with 1.14.9 as the last
+  release supporting Python 2.
+- HoloViews now requires panel >0.13.1
+  ([#4329](https://github.com/holoviz/holoviews/pull/4329))
```

Comment:
- Compatibility note: colormaps for the output of the `datashade` operation have changed to address https://github.com/holoviz/datashader/issues/357 ; see rescale_discrete_levels below.

Reply:
Good suggestion, and it may be worth having a line indicating how to switch `rescale_discrete_levels` back off

Reply:
Addressed in 74942a6cf